src/test/java/edu/hm/ahager/listtest/ListTest.java

```diff
@@ -0,0 +1,82 @@
+package edu.hm.ahager.listtest;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+abstract class ListTest {
+
+    abstract List<Integer> create(int numberOfInitialElements);
+
+    @Test
+    public void testCreateNumberOfInitialElementsSmallerZeroBooms(){
+        // arrange
+        IllegalArgumentException exception = null;
+        // act
+        try {
+            create(-1);
+        }catch (IllegalArgumentException ex){
+            exception = ex;
+        }
+        // assert
+        assertNotNull(exception);
```

Comment:
Mit AssertJ (siehe [Codingstyle](https://github.com/uhafner/codingstyle/blob/master/doc/Testen.md)) verkürzt sich das auf:
```suggestion
       assertThatIllegalArgumentException.isThrownBy(() -> create(-1));
```